src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -219,18 +219,32 @@ typedef struct uct_ib_mlx5_devx_uar {
 
 
 typedef enum {
+    UCT_IB_MLX5_QP_TYPE_UNDEF,
```

Comment:
minor: maybe put it as last, and rename to UCT_IB_MLX5_QP_TYPE_LAST?
this is the convention in other places..
pls fix in next PR..